### PR TITLE
Add backTitle & backIcon to PhotoBrowser's properties.

### PR DESCRIPTION
**Pull request by mokabyls**
This pull request allows you to configure the label of the back button and the icon on back button. If nothing is entered, the default values are kept. **Breaking Change**: the "backImage" property in the TopBar component has been renamed to "backIcon".

### CODE REVIEW

**Comment by mokabyls** [comment]
ping @halilb. Any news about the review ?

File: lib/bar/TopBar.js [hunk 1]
```diff
@@ -19,29 +19,31 @@ export default class TopBar extends React.Component {
...
+    const { onBack, backIcon } = this.props;
 
     // do not display back button if there isn't a press handler
     if (onBack) {
       return (
         <TouchableOpacity style={styles.backContainer} onPress={onBack}>
-          <Image source={backImage} />
-          {Platform.OS === 'ios' &&
+          {backIcon && React.isValidElement(backIcon) && backIcon}
+          {backIcon && !React.isValidElement(backIcon) && <Image source={backIcon} />}
```
Comment (halilb) [inline lib/bar/TopBar.js:41]: We might introduce a new prop called `backElement` to keep this backwards compatible as well:

```suggestion
const useBackElement = !!backElement && React.isValidElement(backElement)

          {useBackElement && backElement}
          {!useBackElement && !!backImage && <Image source={backImage} />}
```

**Review by halilb — COMMENTED** [review]
Thanks for the work @mokabyls! And I'm sorry for reviewing this late.

This looks good except the breaking change part. I commented about a solution we might use by introducing a new prop without removing the old one.

**Comment by mokabyls** [comment]
Hi @halilb,

sorry for the delay. Just made adjustments on the library.
Tell me if you're ok with this modifications.

**Comment by halilb** [comment]
> Hi @halilb,
> 
> sorry for the delay. Just made adjustments on the library.
> Tell me if you're ok with this modifications.

Thank you @mokabyls. It looks good!

**Pull request merged**